builders/pallets-precompiles/pallets/utility.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Utility Pallet
+description: Learn about the available extrinsics for the utility pallet on Moonbeam and how to interact with them using Polkadot.js Apps and the Polkadot.js API.
+keywords: utility, batch, substrate, pallet, moonbeam, polkadot
+---
+
+# The Utility Pallet
+
+![Utility Moonbeam Banner](/images/builders/pallets-precompiles/pallets/utility-banner.png)
+
+## Introduction {: #introduction }
+
+Through Substrate's utility pallet, users on Moonbeam can include multiple calls into a single transaction and use derivative accounts to send calls.
+
+Some of the functionality of the utility pallet is also available through a batch precompile. The precompile is a Solidity interface that enables you to perform batch calls using the Ethereum API. Please refer to the [Batch Precompile](/builders/build/precompiles/batch){target=_blank} guide for more information.
```

Comment:
Remember what I mentioned that the batch precompile has nothing to do with the `utility` pallet